src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -187,7 +197,9 @@ private NeoModule makeSwerveModule(int driveId, int steerId) {
   }
 
   private void configureBindings() {
-    m_driveController.start().onTrue(m_poseEstimator.resetPose(new Pose2d()));
+    m_driveController
+        .start()
+        .onTrue(new InstantCommand(() -> m_poseEstimator.resetPose(new Pose2d())));
```

Comment:
Alternatively there can be a `resetPoseCommand` in addition to `resetPose` that uses the `runOnce` factory.